package.json

```diff
@@ -3,7 +3,29 @@
   "title": "jQuery",
   "description": "JavaScript library for DOM operations",
   "version": "4.0.0-pre",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "node": {
+        "module": "./dist/jquery.mjs",
+        "import": "./dist/jquery-node-wrapper.mjs",
+        "default": "./dist/jquery.js"
+      },
+      "browser": "./dist/jquery.mjs"
+    },
+    "./slim": {
+      "node": {
+        "module": "./dist/jquery.slim.mjs",
+        "import": "./dist/jquery-node-wrapper.slim.mjs",
+        "default": "./dist/jquery.slim.js"
+      },
+      "browser": "./dist/jquery.slim.mjs"
+    },
```

Comment:
This part mostly corresponds with a recommendation from Webpack from:
https://webpack.js.org/guides/package-exports/#providing-commonjs-and-esm-version-stateful